tgui/packages/tgui/interfaces/Orbit.tsx

```diff
@@ -287,6 +289,13 @@ export const Orbit = (props: any, context: any) => {
           source={misc}
           searchText={searchText}
         />
+
+        <BasicSection
+          title="Vehicles"
+          source={vehicles}
+          searchText={searchText}
+        />
+
```

Comment:
1. move this under humans since it's more relevant

2. replace with

```suggestion

         <Section title={`Vehicles - (${vehicles.length})`}>
          {vehicles
            .filter(searchFor(searchText))
            .sort(compareNumberedText)
            .map(thing => (
              <OrbitedButton
                key={thing.name}
                color="blue"
                thing={thing} />
            ))}
        </Section>
```
this code should work I apologise if it does not

this will make it so numbers display properly etc